inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+# Inline Cache with GemFire
+
+Using an inline cache with a database can help improve performance, reduce database scaling limits, and provide greater flexibility in terms of data retrieval. 
```

Comment:
It would be in this introduction to define what an inline cache is and what it does.

Reply:
Would would be in this introduction? I don't understand.

Reply:
Sorry, not sure what happened here.  
It would be good to include a definition of inline caching, what it does and why users would use it, in the introduction.

Reply:
I added more information on what an inline cache is

Reply:
I guess to just differentiate it a bit from the Cache-Aside pattern.  What makes an inline cache different is that the cache  is placed between the application and the database...as opposed to cache aside, where the application is connected to both the cache and the database.